pulsar-broker/src/test/resources/authentication/tls/cacert.pem

```diff
@@ -1,62 +1,29 @@
-Certificate:
-    Data:
-        Version: 3 (0x2)
-        Serial Number:
-            88:08:98:b3:13:d8:00:94
-        Signature Algorithm: sha1WithRSAEncryption
-        Issuer: C=US, ST=CA, O=Apache, OU=Pulsar Incubator, CN=localhost
-        Validity
-            Not Before: Feb 17 01:37:33 2018 GMT
-            Not After : Feb 16 01:37:33 2021 GMT
```

Comment:
I'm not familiar with TLS certificate but just have a question. The original `Certificate - Data - Validity` block has the `Not After` field that might lead to the broken CI. After this change, should we also keep the header-like  block to record the `Not After` time?

I think the root cause is #1244 that intended to create certs which will expire after year 2030. However, the `cacert.pem` still expired after 2021-02-16.